wiki/Glossary/en.md

```diff
@@ -267,13 +270,15 @@ In osu!catch, once a comboset is completed, the fruit will jump off of the plate
 
 The cookie is another word for the _osu!_ logo.
 This has appeared many times in the osu!client.
-The cookie is seen in the main menu, in solo mode, and previously in old crash reports (seen [here](img/Pippi_corruption.jpg)).
+The cookie is seen in the main menu, in solo mode, and previously in old crash reports as seen below.
+
+![](img/Pippi_corruption.jpg "Pippi! Get that cookie out of your mouth - it could be dirty...")
```

Comment:
using a different alt text
`![Window dialog of osu! crashing](img/Pippi_corruption.jpg "Pippi! Get that cookie out of your mouth - it could be dirty...")`